### ISSUE TITLE
serve.sh script can not be located

### ISSUE THREAD

**Issue by kander** [Swader/homestead_improved#1]
Hi Bruno, as mentioned on Twitter yesterday:

~~Ever since the Homestead box was updated to 2.2 (Vagrant automatically updates boxes via "Vagrant Atlas")~~, I'm running into an issue where provisioning fails with the following error:

Update: forced Vagrant to use the 0.2.0 box, problem persisted, so initial hunch regarding the cause of this issue was incorrect.

```
==> default: Running provisioner: shell...
    default: Running: inline script
==> default: stdin: is not a tty
==> default: bash: /vagrant/scripts/serve.sh: No such file or directory
The SSH command responded with a non-zero exit status. Vagrant assumes that this means the command failed. The output for this command should be in the log above. Please read the output to determine what went wrong.
```

When logging in via SSH I can confirm this. The "scripts" directory is not mounted in `/vagrant/scripts`, but in `/vagrant/Code/scripts`: 

```
vagrant@homestead:~$ ls
Code
vagrant@homestead:~/Code$ ls
aliases Homestead.yaml  readme.md  scripts  Vagrantfile
```

Versions:

Running `vagrant box list` shows the following (relevant) boxes:
```
laravel/homestead      (virtualbox, 0.2.0)
laravel/homestead      (virtualbox, 0.2.2)
```

I'm on Vagrant 1.7.1, Virtualbox 4.3.18.

**Comment by PrateekVarshney** [comment]
+1 Same Issue on my machine too.

**Comment by Alain1405** [comment]
Same here!

**Comment by peter-mw** [comment]
Hi i have the same problem

**Comment by Swader** [comment]
I don't understand why I wasn't notified of this issue, sorry all.
Just to confirm, are you all on Windows? 

The problem is generally avoided by putting an absolute path instead of the `.` in the mapping in `Homestead.yaml` on systems that get confused by this (Windows < 8 and OS X). I need to confirm that the fix from @codeorganic works on Linux and OS X in that case, but if it does, I'll merge and it should be fixed.

**Comment by peter-mw** [comment]
Yes i am on Windows 7

**Comment by allenhurff** [comment]
@Swader +1 Could you go ahead and merge your fix [356c98b](https://github.com/codeorganic/homestead_improved/commit/356c98b073727d9012096cb1948e3798653408d1) into your master branch.  It fixed the issue as you suggested it would.

For the record, I'm not running on Windows.  I'm running OS X Yosemite Version 10.10.4 on a Macbook Pro (Retina, 15-inch, Mid 2014).

:shit: Here is what the issue looked like on the 'vagrant up'  BEFORE I made the code changes @Swader suggested in [356c98b](https://github.com/codeorganic/homestead_improved/commit/356c98b073727d9012096cb1948e3798653408d1):

![iterm](https://cloud.githubusercontent.com/assets/209308/7920939/da628368-0856-11e5-9640-b57e2fe9db62.jpg)

:thumbsup: Here is what the issue looked like on the 'vagrant up' AFTER I made the code changes @Swader suggested in [356c98b](https://github.com/codeorganic/homestead_improved/commit/356c98b073727d9012096cb1948e3798653408d1)

![iterm 2](https://cloud.githubusercontent.com/assets/209308/7920942/e05a7848-0856-11e5-833b-0f134c68c99b.jpg)

Now, I just need to [setup my first project on my vagrant](http://laravel.io/forum/06-04-2014-no-input-file-specified-using-homestead)

**Comment by Swader** [comment]
Alright, merged.

Still weird. I clone new projects every day, and never see this problem. Though I use 

```
sed -i '' "s@map\: \.@map\: $PWD@g" Homestead.yaml
```

immediately after cloning, so my folders are mapped to the current working folder...
